engines/python/src/main/java/ai/djl/python/engine/PyModel.java

```diff
@@ -181,24 +181,6 @@ public void load(Path modelPath, String prefix, Map<String, ?> options) throws I
         }
         pyEnv.setEntryPoint(entryPoint);
 
-        Map<String, String> initParameters = pyEnv.getInitParameters();
-        String s3Url = initParameters.get("s3url");
-        String modelId = pyEnv.getInitParameters().get("model_id");
-        if (s3Url != null) {
-            // s3url is deprecated, use model_id instead
-            if (modelId != null) {
-                throw new IllegalArgumentException("model_id and s3url could not both set!");
-            }
-            modelId = s3Url;
-        }
-        if (modelId != null && modelId.startsWith("s3://")) {
-            logger.info("S3 url found, start downloading from {}", modelId);
```

Comment:
Where do you migrate this logic? In the API package?

Reply:
This PR depends on https://github.com/deepjavalibrary/djl-serving/pull/594

the S3 down is moved into ModelInfo